problems/4.divide.js

```diff
@@ -17,19 +19,23 @@ function divideTwo() {
  * @param {number} b - The second number, to be divided by
  * @returns {number} - A whole number equal to a divided by b
  */
-function divideTwoRound() {
+// function divideTwoRound(a , b) {
+//     return Math.round(a / b)
+// }
 
-}
+const divideTwoRound = (a , b) => {return Math.round(a / b)}
 
 /** 
  * Divides a number by zero
  * 
  * @param {number} n - Any number
  * @returns {Infinity} - Equal to Infinity
 */
-function divideByZero() {
+// function divideByZero(n) {
+//     return n / 0
+// }
 
-}
+const divideByZero = (n) => {return n / 0}
```

Comment:
If going to do one line, just make it an implicit return. No return and no curly braces. If using curly braces make it multiple lines.